crypto/src/mls/conversation/own_commit.rs

```diff
@@ -11,35 +11,46 @@ use openmls::prelude::{
 };
 
 impl MlsConversation {
-    pub(crate) fn maybe_self_member_commit<'a>(
+    /// Extract the confirmation tag from a public message that is an own commit.
+    ///
+    /// # Arguments
+    ///
+    /// * `own_commit`: The own commit.
+    ///
+    /// returns: Result<&ConfirmationTag, CryptoError>: The confirmation tag from the own commit or an error if the message is not an own commit.
```

Comment:
I think this last line and the whole `Arguments` area is unnecessary.
The former because we already have the function signature and we already describe what the function does at the top. The latter because it doesn't add any additional information on top of the already well-named argument to the function.

So I think just keeping the first line (14) should be sufficient in this case.
In general, I find the Rust's stdlib a good model for documentation, see e.g.
https://doc.rust-lang.org/std/string/struct.String.html#method.find